dynawo/sources/Models/Modelica/Dynawo/Electrical/Loads/LoadAlphaBetaRestorative.mo

```diff
@@ -33,10 +33,19 @@ model LoadAlphaBetaRestorative "Generic model of a restorative Alpha-Beta load."
 
   equation
     if (running.value) then
-      tFilter * der(UFilteredRawPu) = ComplexMath.'abs' (terminal.V) - UFilteredRawPu;
+
       UFilteredPu = if UFilteredRawPu >= UMaxPu then UMaxPu elseif UFilteredRawPu <= UMinPu then UMinPu else UFilteredRawPu;
-      PPu = PRefPu.value * ((ComplexMath.'abs' (terminal.V) / UFilteredPu) ^ Alpha);
-      QPu = QRefPu.value * ((ComplexMath.'abs' (terminal.V) / UFilteredPu) ^ Beta);
+
+      if (ComplexMath.'abs' (terminal.V)==0) then
```

Comment:
Normally, UFilteredPu should be looked at too as we divide by it. I don't remember why it didn't work with UFilteredPu in the if, instead of ComplexMath.'abs' (terminal.V). Can we try again ?
If it doesn't work, I suggest we put if (ComplexMath.'abs' (terminal.V) == 0) and (UFilteredPu == 0) then